source/_lovelace/button.markdown

```diff
@@ -11,10 +11,33 @@ The Button card allows you to add buttons to perform tasks.
 Screenshot of the Button card.
 </p>
 
-```yaml
-type: button
-entity: light.living_room
-```
+## Visual Editor
```

Comment:
```suggestion
## Card Settings
```

Reply:
Technically, all these variables (including the YAML configuration variables) are card settings - in one case we are just changing them in a visual editor, and in another case we are changing them either in the code editor using YAML or in YAML mode.  Could we maybe have something like this?
![90C31047-D9B3-4CE1-9F81-CB55CAA8A6A9](https://user-images.githubusercontent.com/64062738/88123637-69d76e00-cc1f-11ea-98fc-ee26a0f39995.jpeg)
![0BA85689-B965-494C-B873-D2ACB8C1556D](https://user-images.githubusercontent.com/64062738/88123646-6e9c2200-cc1f-11ea-835f-36a1d01225f8.jpeg)

Reply:
I do like the `Code Editor or YAML Mode`

But I still don't think we need Visual Editor. To a user coming in they don't know that they are using a visual editor. They just know they are having trouble with the card. The Card Settings are for both and explain the card as a whole. Therefore the first thing a user sees should be the settings covering the card. Even if they are using YAML mode. They may not even need to scroll down if they just need a quick explanation on the setting as the Card Settings are for both

Reply:
Yea that makes sense. I’ll leave in the Code Editor heading and remove the Visual Editor bit.